steps/browserstack-bitrise-xcui-step/1.0.0/step.yml

```diff
@@ -0,0 +1,231 @@
+    description: Video recording helps you view the entire test execution and helps
+      debug any failed test.
+    summary: Record video of the test execution
+    title: Video recording
+    value_options:
+    - true
+    - false
+  video_recording: true
+- opts:
+    category: Test Configuration
+    description: A project name helps you group multiple XCUITest test executions
+      under a single project.
+    summary: Project name of the tests.
+    title: Project name
+  project: null
+- opts:
+    category: Test Configuration
+    description: |
+      A callback URL helps you enable BrowserStack notify you upon the completion of all builds under a given project. You will receive a callback if no new builds triggered for 5 minutes and all existing builds have completed their execution.
+
+      This parameter requires that the **Project name** parameter also be set.
+    summary: A callback URL to enable BrowserStack notify about completion of build
+      under a given project.
+    title: Notify project status
+  project_notify_url: null
+- opts:
+    category: Test Configuration
+    description: Enable local testing to retrieve app data hosted on local/private
+      servers
+    summary: Enable local testing to retrieve app data hosted on local/private servers
+    title: Local testing
+    value_options:
+    - true
+    - false
+  use_local: false
+- opts:
+    category: Test Configuration
+    description: |
+      Enable test sharding to split tests cases into different groups instead of running them sequentially.
+      Add the sharding value json here. Examples:
+      Input for only-testing strategy:
+      {"numberOfShards": 2, "mapping": [{"name": "Shard 1", "strategy": "only-testing", "values": ["SampleXCUITestsClass/testAlert", "SampleXCUITestsClass/testText"]}, {"name": "Shard 2", "strategy": "only-testing", "values": ["SampleXCUITestsClass/testLogin"]}]}
+
+      Input for skip-testing strategy:
+      {"numberOfShards": 2, "mapping": [{"name": "Shard 1", "strategy": "skip-testing", "values": ["SampleXCUITestsClass/testAlert"]}, {"name": "Shard 2", "strategy": "skip-testing", "values": ["SampleXCUITestsClass/testText"]}]}
+    summary: Test sharding splits and run test cases into different group called shards.
+      This behaviour is helpful when your test-suite has a large number of test cases.
+      [Learn more.](https://www.browserstack.com/docs/app-automate/xcuitest/test-sharding)
+    title: Test sharding
+  use_test_sharding: null
+- filter_test: null
+  opts:
+    category: Test Configuration
+    description: |
+      Provide a comma-separated list of class or test names followed by supported filters: "only-testing" and "skip-testing".
+
+      For example: `only-testing SampleXCUITestsClass/testAlertTest`
+
+      [Learn more](https://www.browserstack.com/docs/app-automate/xcuitest/select-test-cases)
+    summary: "Provide a comma-separated list of class or test names followed by supported
```

Comment:
This summary should be a bit shorter, ideally a single sentence.

Reply:
"Provide a comma-separated list of class or test names followed by supported [filtering strategies](https://www.browserstack.com/docs/app-automate/xcuitest/select-test-cases).